ruby-challenges.rb

```diff
@@ -10,22 +10,70 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def find_words_with_letter(words, letter)
+    result = []
+    words.each do |word|
+      if word.include?(letter)
+        result.push(word)
+      end
+    end
+    return result
+  end
+  
```

Comment:
Sweet! How could you refactor this to not need an empty array to push things into?